samples/core/tfx-oss/TFX Example.ipynb

```diff
@@ -89,179 +94,12 @@
-    "  statistics_gen = StatisticsGen(input_data=example_gen.outputs.examples)\n",
-    "\n",
-    "  # Generates schema based on statistics files.\n",
-    "  infer_schema = SchemaGen(stats=statistics_gen.outputs.output)\n",
-    "\n",
-    "  # Performs anomaly detection based on statistics and data schema.\n",
-    "  validate_stats = ExampleValidator(\n",
-    "      stats=statistics_gen.outputs.output, schema=infer_schema.outputs.output)\n",
-    "\n",
-    "  # Performs transformations and feature engineering in training and serving.\n",
-    "  transform = Transform(\n",
-    "      input_data=example_gen.outputs.examples,\n",
-    "      schema=infer_schema.outputs.output,\n",
-    "      module_file=_taxi_utils)\n",
-    "\n",
-    "  # Uses user-provided Python function that implements a model using TF-Learn.\n",
-    "  trainer = Trainer(\n",
-    "      module_file=_taxi_utils,\n",
-    "      transformed_examples=transform.outputs.transformed_examples,\n",
-    "      schema=infer_schema.outputs.output,\n",
-    "      transform_output=transform.outputs.transform_output,\n",
-    "      train_args=trainer_pb2.TrainArgs(num_steps=10000),\n",
-    "      eval_args=trainer_pb2.EvalArgs(num_steps=5000),\n",
-    "      custom_config={'cmle_training_args': _cmle_training_args})\n",
-    "\n",
-    "  # Uses TFMA to compute a evaluation statistics over features of a model.\n",
-    "  model_analyzer = Evaluator(\n",
-    "      examples=example_gen.outputs.examples,\n",
-    "      model_exports=trainer.outputs.output,\n",
-    "      feature_slicing_spec=evaluator_pb2.FeatureSlicingSpec(specs=[\n",
-    "          evaluator_pb2.SingleSlicingSpec(\n",
-    "              column_for_slicing=['trip_start_hour'])\n",
-    "      ]))\n",
-    "\n",
-    "  # Performs quality validation of a candidate model (compared to a baseline).\n",
-    "  model_validator = ModelValidator(\n",
-    "      examples=example_gen.outputs.examples, model=trainer.outputs.output)\n",
-    "\n",
-    "  # Checks whether the model passed the validation steps and pushes the model\n",
-    "  # to a file destination if check passed.\n",
-    "  pusher = Pusher(\n",
-    "      model_export=trainer.outputs.output,\n",
-    "      model_blessing=model_validator.outputs.blessing,\n",
-    "      custom_config={'cmle_serving_args': _cmle_serving_args},\n",
-    "      push_destination=pusher_pb2.PushDestination(\n",
-    "          filesystem=pusher_pb2.PushDestination.Filesystem(\n",
-    "              base_directory=_serving_model_dir)))\n",
-    "\n",
-    "  return [\n",
-    "      example_gen, statistics_gen, infer_schema, validate_stats, transform,\n",
-    "      trainer, model_analyzer, model_validator, pusher\n",
-    "  ]\n",
-    "\n",
-    "\n",
-    "pipeline = KubeflowDagRunner().run(_create_pipeline())"
+    "%load https://raw.githubusercontent.com/tensorflow/tfx/master/tfx/examples/chicago_taxi_pipeline/taxi_pipeline_kubeflow_gcp.py"
```

Comment:
Curious how this line will work. After loading the python file shall we run it in some way so that `chicago_taxi_pipeline_kubeflow.tar.gz` can be used to submit for running?

Reply:
Yes, this is how the leading sentences have instructed, but there isn't actually `%load` in the cell.

After `%load`, one is supposed to update `_input_bucket` and such, and execute the cell once again.

Reply:
Sounds good.